### PR TITLE
#503: Python 3.8 cuda conda minimal

### PR DESCRIPTION
**Pull request by tkilias**

### CODE REVIEW

File: flavors/template-Exasol-all-python-3.8-cuda-conda/flavor_base/build_deps/packages/apt_get_packages [hunk 1]
```diff
@@ -1,9 +1,9 @@
 coreutils|8.28-1ubuntu1
-locales|2.27-3ubuntu1.6
```
Comment (tomuben) [inline flavors/template-Exasol-all-python-3.8-cuda-conda/flavor_base/build_deps/packages/apt_get_packages:2]: what is the reason to go back to an older version?

File: flavors/template-Exasol-all-python-3.8-cuda-conda/flavor_base/build_deps/packages/conda_packages [hunk 2]
```diff
@@ -0,0 +1,7 @@
+protobuf|3.20.1
+pcre|8.45
+chrpath|0.16
+bazel|5.1.1
```
Comment (tomuben) [inline flavors/template-Exasol-all-python-3.8-cuda-conda/flavor_base/build_deps/packages/conda_packages:4]: maybe already go to 5.2.0

**Review by tomuben — COMMENTED** [review]
check my comments

File: flavors/template-Exasol-all-python-3.8-cuda-conda/flavor_base/build_deps/packages/apt_get_packages (hunk 1 above)
Comment (tkilias) [reply flavors/template-Exasol-all-python-3.8-cuda-conda/flavor_base/build_deps/packages/apt_get_packages]: probably a error and this file should get deleted anyway

**Review by tomuben — APPROVED** [review]

**Pull request merged**